tools/hangar/src/cli.test.ts

```diff
@@ -99,36 +91,21 @@ async function enterTestDir(testDir: string) {
   $.cwd = testDir;
 }
 
-enum InvocationType {
-	Direct,
-	NPX,
-}
+async function runWingCommand(command: string[], wingFile: string) {
+  const isError = path.dirname(wingFile).endsWith("error");
 
-async function runWingCommand(type: InvocationType, command: string[], wingFile: string) {
-	const isError = path.dirname(wingFile).endsWith("error");
+  const cmd: string[] = [npxBin, "winglang", ...command, wingFile];
```

Comment:
Why not invoke the winglang bin directly? Are we trying to test `npx`-related behavior, or are the tests faster this way, or is there another reason?

Reply:
Yeah good point. I think initially there was an intent to test workflows, and npx is probably a common one. but since hangar installs the cli, npx is really not doing much. npx is just slower.

Changed it to direct invocation.